client/me/purchases/paths.js

```diff
@@ -1,53 +1,35 @@
 /** @format */
-function purchasesRoot() {
-	return '/me/purchases';
-}
 
-function addCreditCard() {
-	return purchasesRoot() + '/add-credit-card';
-}
+export const purchasesRoot = '/me/purchases';
 
-function billingHistory() {
-	return purchasesRoot() + '/billing';
-}
+export const addCreditCard = purchasesRoot + '/add-credit-card';
```

Comment:
It seems there is an occurrence of `addCreditCard` that still calls it as a function in https://github.com/Automattic/wp-calypso/blob/master/client/me/purchases/credit-cards/index.jsx#L52 that needs to be updated too.

Currently, it'll cause an error (to reproduce, you can visit http://calypso.localhost:3000/me/purchases/billing and click on the "Add Credit Card" button).

Reply:
See previous comment - updated in 6de976f800